DolbyIO/Source/Private/DolbyIOSubsystem.cpp

```diff
@@ -241,6 +241,22 @@ void UDolbyIOSubsystem::Initialize(const FString& Token)
 	Sdk->conference()
 	    .add_event_handler([this](const dolbyio::comms::conference_status_updated& Event)
 	                       { UpdateStatus(Event.status); })
+	    .then(
+	        [this](dolbyio::comms::event_handler_id)
+	        {
+		        return Sdk->conference().add_event_handler(
+		            [this](const dolbyio::comms::participant_added& Event)
+		            {
+			            const FDolbyIOParticipantInfo Info = ToUnrealParticipantInfo(Event.participant);
+						auto Status = Event.participant.status;
+			    		DLB_UE_LOG(Log, "Participant added event: UserID=%s Name=%s ExternalID=%s Status=%s",
+			                       *Info.UserID, *Info.Name, *Info.ExternalID, Status ? *ToString(*Status) : TEXT("not set"));
+				        if (!Status || *Status != dolbyio::comms::participant_status::left)
```

Comment:
I would just do an early return at the very beginning:

```
if (Event.participant.status && *Event.participant.status == dolbyio::comms::participant_status::left)
{
    return;
}
```

I don't see the reason to convert to FDolbyIOParticipantInfo or log the "past" participants and it can be confusing to the user to see that log but not the event. Especially if it is a long-running conference with hundreds of past users.

It will also be consistent with participant_updated below.

Reply:
well it is consistent with participant_updated below in terms both log and not raise an event for some statuses. Also it is only temporary solution and we need to agree details directly.